backend/__mock__/getReported.js

```diff
@@ -0,0 +1,14 @@
+//mocked module (stores reported users)
+
+function getReported() {
+    var reportedUsers = [];
+    reportedUsers.push("aaa@gmail.com");
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/karatwan/CPEN321_world_resources/pullRequest?prid=9811192)
```suggestion
  reportedUsers.push("aaa@gmail.com");
```